include/freerdp/settings.h

```diff
@@ -1441,6 +1441,9 @@ struct rdp_settings
 	 */
 
 	/* Extensions */
+	ALIGN64 BOOL WmSkipTaskbar[2];
+	ALIGN64 BOOL WmSkipPager[2];
```

Comment:
I'd personally prefer these to be named something like UseWmSkipPager and WmSkipPagerEnabled instead of using arrays. Because

```c
if (settings->UseWmSkipPager)
        SetWmSkipPager(settings->WmSkipPagerEnabled);
```

is arguably more readable and unambiguous than

```c
if (settings->WmSkipPager[0])
        SetWmSkipPager(settings->WmSkipPager[1]);
```

Also, I think that the `Extensions` comment refers to the two fields that follow, so it isn't nice to separate them. The new settings can be placed at the end of the structure.

Reply:
I've done at last commit, I changed with WmSkipPager and WmSkipPagerEnabled.